src/gui/transferlistmodel.cpp

```diff
@@ -55,6 +55,73 @@ static QIcon getErrorIcon();
 
 static bool isDarkTheme();
 
+TransferListTextColors::TransferListTextColors()
+{
+    m_unknownTextColor = getColorByState(BitTorrent::TorrentState::Unknown);
+    m_forcedDownloadingTextColor = getColorByState(BitTorrent::TorrentState::ForcedDownloading);
+    m_downloadingTextColor = getColorByState(BitTorrent::TorrentState::Downloading);
+    m_downloadingMetadataTextColor = getColorByState(BitTorrent::TorrentState::DownloadingMetadata);
+    m_allocatingTextColor = getColorByState(BitTorrent::TorrentState::Allocating);
+    m_stalledDownloadingTextColor = getColorByState(BitTorrent::TorrentState::StalledDownloading);
+    m_forcedUploadingTextColor = getColorByState(BitTorrent::TorrentState::ForcedUploading);
+    m_uploadingTextColor = getColorByState(BitTorrent::TorrentState::Uploading);
+    m_stalledUploadingTextColor = getColorByState(BitTorrent::TorrentState::StalledUploading);
+    m_checkingResumeDataTextColor = getColorByState(BitTorrent::TorrentState::CheckingResumeData);
+    m_queuedDownloadingTextColor = getColorByState(BitTorrent::TorrentState::QueuedDownloading);
+    m_queuedUploadingTextColor = getColorByState(BitTorrent::TorrentState::QueuedUploading);
+    m_checkingUploadingTextColor = getColorByState(BitTorrent::TorrentState::CheckingUploading);
+    m_checkingDownloadingTextColor = getColorByState(BitTorrent::TorrentState::CheckingDownloading);
+    m_pausedDownloadingTextColor = getColorByState(BitTorrent::TorrentState::PausedDownloading);
+    m_pausedUploadingTextColor = getColorByState(BitTorrent::TorrentState::PausedUploading);
+    m_movingTextColor = getColorByState(BitTorrent::TorrentState::Moving);
+    m_missingFilesTextColor = getColorByState(BitTorrent::TorrentState::MissingFiles);
+    m_errorTextColor = getColorByState(BitTorrent::TorrentState::Error);
+}
+
+QColor TransferListTextColors::textColorByState(int state) const
```

Comment:
Why convert `BitTorrent::TorrentState` to `int` and back?

Reply:
to not include `bittorrent/torrenthandle.h` in `transferlistmodel.h`. Should I include it?

Reply:
I think yes.

Reply:
done